protocol/0095-LIQM-liquidity_mechanisms.md

```diff
@@ -21,48 +21,112 @@ The mechanisms also enable the following features:
+A number in the range $[0, 100]$ which defines the amount to scale the quantum of the markets settlement or quote asset by (for derivative and spot markets respectively) when setting the initial quantum of the internal asset representing the ELS points for the market.
 
-At the end of each epoch, each party accrues ELS points for the relevant market as follows:
+Note, this parameter influences the minimum amount of ELS points a party will have to accrue before being able to transfer.
 
-$$ELS_{i_j} = V_{i_j} \cdot 10^{-Q}$$
+### `liquidity.els.transferEnabled`
 
-Where:
+A boolean, defaulting to `true`, which sets whether internal assets representing ELS points are allowed to be transferred when the asset is first created.
 
-- $ELS_{j}$ is the ELS points accrued by party $j$ in epoch $i$
-- $V_{i_j}$ is the notional maker fees of party $j$ in epoch $i$
-- $Q$ is the quantum of the asset in which the markets prices are expressed in (settlement for future and perpetual markets, quote for spot markets)
+Note, the restriction on the asset can always be updated later through governance.
 
-> [!NOTE]
-> the maker fees is scaled by the assets quantum such that 1 ELS point is earned for approx. every 1 USD of maker fees received.
+### `liquidity.els.settlementAssetEnabled`
+
+A boolean, defaulting to `false`, which sets whether internal assets representing ELS points are allowed to be used as the settlement asset of a derivative market when they are first created.
+
+Note, the restriction on the asset can always be updated later through governance.
+
+### `liquidity.els.baseAssetEnabled`
+
+A boolean, defaulting to `false`, which sets whether internal assets representing ELS points are allowed to be used as the base asset of a spot market when they are first created.
 
-## Designating Liquidity Providers
+Note, the restriction on the asset can always be updated later through governance.
 
-A party will only be able to receive liquidity fees or earn liquidity rewards providing they are designated as an LP for that epoch.
+### `liquidity.els.quoteAssetEnabled`
 
-A party will only be designated as an LP providing they received more than a specified proportion of the markets total maker fees in the previous epoch, let this requirement be $N$ (the network parameter `liquidityProvider.proportionRequirement`). Throughout the epoch the network will track each parties maker fees, $M$.  At the end of epoch $i$, a party will be designated as an LP for epoch $i+1$ providing:
+A boolean, defaulting to `false`, which sets whether internal assets representing ELS points are allowed to be used as the quote asset of a spot market when they are first created.
+
+Note, the restriction on the asset can always be updated later through governance.
+
+## Mechanisms
+
+There are three liquidity mechanisms that take place at the end of the epoch.
+
+1. Liquidity fees are distributed amongst liquidity providers
+2. New liquidity providers are delegated
```

Comment:
Perhaps "designated" rather than "delegated" to remove any possible association with delegation of tokens?